shenyu-examples/shenyu-examples-http/src/main/java/org/apache/shenyu/examples/http/controller/HttpTestController.java

```diff
@@ -367,4 +367,30 @@ private UserDTO buildUser(final String id, final String name) {
         userDTO.setUserName(name);
         return userDTO;
     }
+
+    /**
+     * the waf pass.
+     *
+     * @return response. result bean
+     */
+    @GetMapping ("/hystrix/pass")
+    public ResultBean hystrixPass() {
```

Comment:
IMO, we can reuse existing endpoint, no need to add new ones, WDYT?

Reply:
https://github.com/apache/incubator-shenyu/blob/e3456e91a093057455fbf09916cca09a3982e9ca/shenyu-examples/shenyu-examples-http/src/main/java/org/apache/shenyu/examples/http/controller/HttpTestController.java#L159-L164


https://github.com/apache/incubator-shenyu/blob/e3456e91a093057455fbf09916cca09a3982e9ca/shenyu-examples/shenyu-examples-http/src/main/java/org/apache/shenyu/examples/http/controller/HttpTestController.java#L204-L212

I noticed that the previous test also created a corresponding method, so I also created a new method belonging to the Hystrix plugin. I think doing this reduces the coupling between methods.

Reply:
We should probably unify and simplify these endpoints instead of making them more complex and repetitive. . . Of course, you can also do this in subsequent prs.

Reply:
I think there should be separate controllers for each type of test.
When integrated, a single change affects all tests.
I think the unified approach is suitable for situations that don't change, like ResultBean, only code, message, data .